packages/internet_detector.vm/tools/default.ini

```diff
@@ -0,0 +1,352 @@
+# * DNSResponse      - IP address to respond with for A record DNS queries. (DNSListener)
+# * NXDomains        - A number of DNS requests to ignore to let the malware cycle through
+#                      all of the backup C2 servers. (DNSListener)
+# * Banner           - FTPListener, IRCListener: FTP or IRC banner to display.
+#                      Valid settings are any banner string, or !<key> where
+#                      <key> is a valid key in the BANNERS dictionary within
+#                      FTPListener.py or IRCListener.py, or !random to
+#                      randomize among the banners in the BANNERS dictionary.
+#                      The default value if none is specified is !generic,
+#                      which selects the banner in the BANNERS dictionary going
+#                      by that key. Banner string may specify the following
+#                      escapes/insertions:
+#                          {servername} - ServerName setting value
+#                          {tz} - Time zone, currently hard-coded to 'UTC'
+# * ServerName       - FTPListener, IRCListener: FTP or IRC server name for
+#                      insertion into selected default banners or into a
+#                      user-specified banner string. Valid settings are any
+#                      hostname string, !hostname to insert the actual hostname
+#                      of the system, or !random to generate a random hostname
+#                      between 1 and 15 characters (inclusive).
+
+[ProxyTCPListener]
+Enabled:     True
+Protocol:    TCP
+Listener:    ProxyListener
+Port:        38926
+Listeners:   HTTPListener, RawListener, FTPListener, DNSListener, POPListener, SMTPListener, TFTPListener, IRCListener
+Hidden:      False
+
+[ProxyUDPListener]
+Enabled:     True
+Protocol:    UDP
+Listener:    ProxyListener
+Port:        38926
+Listeners:   RawListener, DNSListener, TFTPListener, FTPListener
+Hidden:      False
+
+[Forwarder]
+Enabled:     False
+Port:        8080
+Protocol:    TCP
+ProcessWhiteList: chrome.exe
+Hidden:      False
+
+[RawTCPListener]
+Enabled:     True
+Port:        1337
+Protocol:    TCP
+Listener:    RawListener
+UseSSL:      No
+Timeout:     10
+Hidden:      False
+# To read about customizing responses, see docs/CustomResponse.md
+# Custom:    sample_custom_response.ini
+
+[RawUDPListener]
+Enabled:     True
+Port:        1337
+Protocol:    UDP
+Listener:    RawListener
+UseSSL:      No
+Timeout:     10
+Hidden:      False
+# To read about customizing responses, see docs/CustomResponse.md
+# Custom:    sample_custom_response.ini
+
+[FilteredListener]
+Enabled:     False
+Port:        31337
+Protocol:    TCP
+Listener:    RawListener
+UseSSL:      No
+Timeout:     10
+ProcessWhiteList: ncat.exe, nc.exe
+HostBlackList: 5.5.5.5
+Hidden:      False
+
+[DNS Server]
+Enabled:     True
+Port:        53
+Protocol:    UDP
+Listener:    DNSListener
+ProcessBlackList: pythonw.exe
```

Comment:
we can not exclude `pythonw.exe`. It is too generic and it could be needed to run malware.

Two options:
### duplicate `pythonw.exe` with a different name [ugly temporary workaround]
In the package, duplicate `pythonw.exe` in the same directory with name `pythonw_internet_detector.exe`. Use `pythonw_internet_detector.exe` to run the tool and in the fakenet log. It is a bit ugly, but we could use it as a temporary workaround to be able to merge this PR and start testing this tool.

### build the tool
We could build the tool into an executable in the package. I see [capa doing just: `python -m build`](https://github.com/mandiant/capa/blob/master/.github/workflows/publish.yml), so it may be also not difficult to do. @mr-tz and @williballenthin may be able to help with how to best implement it. :pray: